src/Marten/Schema/JsonLocatorField.cs

```diff
@@ -14,38 +15,55 @@ public static JsonLocatorField For<T>(EnumStorage enumStyle, Casing casing, Expr
         {
             var property = ReflectionHelper.GetProperty(expression);
 
-            return new JsonLocatorField("d.data", new StoreOptions(), enumStyle, casing, property);
+            return new JsonLocatorField("d.data", null, enumStyle, casing, property);
         }
 
         private readonly Func<Expression, object> _parseObject = expression => expression.Value();
 
-        public JsonLocatorField(string dataLocator, StoreOptions options, EnumStorage enumStyle, Casing casing, MemberInfo member) : base(enumStyle, member)
+        public JsonLocatorField(string dataLocator, string databaseSchemaName, EnumStorage enumStyle, Casing casing, MemberInfo member, string pgType = null) :
```

Comment:
**FIX:** Same comment as in https://github.com/JasperFx/marten/pull/1334/files#r313276676. But here I think that we should keep backward compatibility, so at least keep the constructor with Store options.

Reply:
restored